src/page/ProductAddedModal.page.ts

```diff
@@ -1,11 +1,12 @@
-import { $, ElementFinder, promise } from 'protractor';
+import { $, ElementFinder, promise, browser, ExpectedConditions } from 'protractor';
 
 export class ProductAddedModalPage {
   private get productAddedModal(): ElementFinder {
     return $('[style*="display: block;"] .button-container > a');
   }
 
-  public goToSummary(): promise.Promise<void> {
+  public async goToSummary(): Promise<void> {
+    await browser.wait(ExpectedConditions.elementToBeClickable(this.productAddedModal));
```

Comment:
is a good practice to add a specific time for the explicit waits, like:
```js
browser.wait(ExpectedConditions.elementToBeClickable(this.productAddedModal), 5000);
```

meaning that this will wait 5 seconds as much for the element to be clickable

the value of 5000 varies depending on show is the average time you should wait. The lower the better.